doc/nrf/known_issues.rst

```diff
@@ -150,15 +151,15 @@ CIA-661: Asset Tracker v2 application configured for LwM2M cannot be built for t
 Serial LTE Modem
 ================
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
 
 NCSDK-13895: Build failure for target Thingy:91 with secure_bootloader overlay
   Building the application for Thingy:91 fails if secure_bootloader overlay is included.
 
 Other issues
 ============
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

Comment:
This will be fixed in 2.1.0, FPU support in TF-M.

Reply:
Updated.